scripts/language_model/cache_language_model.py

```diff
@@ -176,7 +177,8 @@ def evaluate(data_source, batch_size, ctx=None):
         The loss on the dataset
     """
     total_L = 0
-    hidden = cache_cell._pretrained_lm_model.begin_state(func=mx.nd.zeros, batch_size=batch_size, ctx=context[0])
+    hidden = cache_cell._pretrained_lm_model.\
```

Comment:
we should avoid using members and methods that are hidden. Let's add an API to CacheCell for `begin_state` that proxies to the underlying language model.